runtime/lua/vim/pack.lua

```diff
@@ -625,7 +625,7 @@ local function pack_add(plug, load)
     local after_paths = vim.fn.glob(plug.path .. '/after/plugin/**/*.{vim,lua}', false, true)
     --- @param path string
     vim.tbl_map(function(path)
-      pcall(vim.cmd.source, vim.fn.fnameescape(path))
+      vim.cmd.source(vim.fn.fnameescape(path))
```

Comment:
off-topic: IIRC `nvim_cmd(..., {magic=''}` was supposed to disable all "magic" in cmdline args, to avoid the need for fnameescape(). Idk if it was implemented

Reply:
> off-topic: IIRC `nvim_cmd(..., {magic=''}` was supposed to disable all "magic" in cmdline args, to avoid the need for fnameescape(). Idk if it was implemented

Maybe, I also don't quite remember. Decided to wait until there are tests.